pkg/cmd/build/build.go

```diff
@@ -457,7 +480,7 @@ func parseTempSecrets(secretTempFolder string, buildOptions *types.BuildOptions)
 		}
 
 		// replace src
-		buildOptions.Secrets[indx] = fmt.Sprintf("%ssrc=%s", splitSecret[0], tmpfile.Name())
```

Comment:
is this another bug?

Reply:
the variables use to form the string are changed now. for grabbing the ID i not use secretID so i changed the string format for the template too. this is part of the bug fix